src/makeRobot.js

```diff
@@ -38,7 +38,69 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'chip version': version,
```

Comment:
Avoid ambiguity here, you don't need to add the same names while building an object
```suggestion
    name,
    wheels,
    version,
```

Reply:
Thank's, fixed.